uiowa/uiowa
# AMCS - extended person type fields

Resolves #3782.



# How to test



As an amcs.uiowa.edu site editor, I want to include additional information for my student person profiles including dissertation topic/thesis title, defense date, current position institution and title, and advisor(s). I want this information to be included as separate fields so that I may create a view page such as https://amcs.prod.drupal.uiowa.edu/students/all-graduates.

As an immuno.grad.uiowa.edu site editor, I want to continue working as if no changes have been made (ie my student person profiles should still contain fields for advisors, defense date, dissertation topic, program start, and current position institution and title).

As a SiteNow V3 site editor who's not using either the immuno.grad.uiowa.edu or grad_person_extended site splits, I want to continue working as if no changes have been made.
